README.md

```diff
@@ -1,2 +1,29 @@
-# community
-Metal3-io governance material.
+# Metal3 community
+
+Welcome to Metal3-io community! This repository contains governance materials
```

Comment:
Title says Metal3 community, while the body says Metal3-io community.

I'd prefer if we were the Metal3 community as the IO part is just from the domain name.

Reply:
> Title says Metal3 community, while the body says Metal3-io community.
> 
> I'd prefer if we were the Metal3 community as the IO part is just from the domain name.

Agree, changed